jkq/ddsim/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .jkqprovider import JKQProvider
+from .circuitsimulator import CircuitSimulator
+from jkq.ddsim import pyddsim
```

Comment:
It should be possible to use `from .pyddsim import *` or whatever classes/methods you want to import from there.
This should allow you to skip the annoying `pyddsim.[...]` whenever calling corresponding methods in Python after including `ddsim` with `from jkq import ddsim`.